cds_ils/cli.py

```diff
@@ -223,6 +227,43 @@ def run_command(command, catch_exceptions=False, verbose=True):
     run_command("vocabulary index opendefinition opendefinition --force")
 
 
+def recreate_cover(pid, record_class):
+    """Recreate cover for a given record pid."""
+    record = record_class.get_record_by_pid(pid)
+    if record:
+        pick_identifier_with_cover_task(record)
+        click.secho("Recreated cover for PID: {}".format(pid), fg="blue")
+    else:
+        click.secho("No document or series found with PID: {}", fg="red")
+
+
+@covers.command()
+@click.option(
+    "-t", "--pid-type", required=True, type=click.Choice(["docid", "serid"])
+)
+@click.option("-a", "--all", required=False, is_flag=True)
+@click.option("-p", "--pid")
+@with_appcontext
+def recreate(pid, pid_type, all):
+    """Recreate covers command."""
+    if pid_type == DOCUMENT_PID_TYPE:
+        record_class = current_app_ils.document_record_cls
+    elif pid_type == SERIES_PID_TYPE:
+        record_class = current_app_ils.series_record_cls
+    if all:
+        pid_list = (
+            PersistentIdentifier.query.filter_by(
+                object_type="rec", status=PIDStatus.REGISTERED
+            )
+            .filter(PersistentIdentifier.pid_type.in_([pid_type]))
```

Comment:
Just a comment, no need to change, just wondering why this `list in` instead of a `==`... it might be more performant too.

Reply:
that's a leftover, I had a list of pid types there before as a an argument. I will change it to equals